include/alpaka/core/Common.hpp

```diff
@@ -127,3 +127,28 @@
 #else
     #define ALPAKA_STATIC_ACC_MEM_CONSTANT
 #endif
+
+
+// ALPAKA_LAMBDA defines compiler dependent host-device annotation and capture
+// - from: https://github.com/kokkos/kokkos/wiki/Lambda-Dispatch
+#if BOOST_COMP_NVCC || BOOST_COMP_CLANG_CUDA
+
+#if BOOST_COMP_NVCC < BOOST_VERSION_NUMBER(8, 0, 0)
```

Comment:
We do not support nvcc versions lower than 8.0 anyways so this can probably be removed.

Reply:
is removed.